packages/codemirror-lang-chordpro/src/syntax.grammar

```diff
@@ -1,18 +1,24 @@
+@dialects { pro, crd }
```

Comment:
@isaiahdahl any suggestions on naming these? I chose `crd` because that's the file extension that I've seen used for songsheets that use chords over lyrics.

Reply:
For PraiseCharts we've always referred to it as "Chords Over Words" in the current editor mode dropdowns and surveys etc. But "Chords Over Words" and "Chords Over Lyrics" are pretty much synonymous, It's just what we've picked to be consistent.

That translated to us just recently naming the new ChordSheetJS parser the ChordsOverWordsParser, and we will be creating the ChordsOverWordsFormatter.

All that to say, is there a reason we want short, extension style dialet labels?

What about:
```
@dialects { chordPro, chordsOverWords }
```
I tend to prefer clarity over shortness when it's not so obvious what the shortform means. But is writing it out like that gonna be too long for cleanliness?